### DISCUSSION TITLE
Batch size, what is optimising?

### DISCUSSION THREAD

**Discussion by JoostvanBruggen** [Mattk70/Chirpity-Electron#95] [General]
The batch size is optimizing the processing speed on largers files. Right.. but please tell us what "optimising" is. If we don't know this we also don't know to lower it or to increase the batch size.  See your explaination at Threads. That is a fine explaination!

**Comment by Mattk70** [comment]
It's about maximising the speed of processing. I'll try to make it clearer. The best value will depend on whether you use the CPU or GPU backend, and also the specifics of the hardware you are using. It really is a case of trial and error. If the batch size is too large, you may see a slow down in processing speed. Generally, 32 is a good figure though.

**Reply by JoostvanBruggen** [reply]
Ok, thank you! Trial and error it is! :-)
